docs(mTLS): Terminology update for mTLS (#7479)
strimzi/strimzi-kafka-operator: documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

kubectl get secret _KAFKA-CLUSTER-NAME_-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
+
Use the extracted certificate in your Kafka client to configure the TLS connection.

> If you enabled any authentication, you will also need to configure SASL or mTLS authentication.

Reviewer: There might be also OAuth etc. I would change this to be more generic.

```suggestion
If you enabled any authentication, you will also need to configure it in your client.
```
